### ISSUE TITLE
Rule proposal:  `no-identical-alternate`

### ISSUE THREAD

**Issue by fisker** [sindresorhus/eslint-plugin-unicorn#2153]
### Description

Enfore clean code.

### Fail

```js
a ? b ? c : 1 : 1
```

### Pass

```js
a && b ? c  : 1
```

**Comment by B2o5T** [comment]
also, it can verify `if`/`if-else` statements

```js
if (foo) {
  return true
}
return true
```
```js
if (foo) {
  return true
} else {
  return true
}
```

**Comment by sindresorhus** [comment]
Accepted
